src/Discord.Net.Interactions/Discord.Net.Interactions.csproj

```diff
@@ -20,7 +20,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="8.0.1" />
```

Comment:
Do we really need to reference the whole package rather than just the `Abstractions` one?

Reply:
Yep, otherwise the attribute cant be found.

Reply:
Nevermind, it seems the type is contained within Abstractions, no idea why I added it

Reply:
![Screenshot_2024-10-12-19-22-25-416_com.android.chrome-edit.jpg](https://github.com/user-attachments/assets/7599c8df-f329-403c-81ae-c16943e8574b)

🤔